includes/class-sensei-learner-profiles.php

```diff
@@ -35,9 +35,28 @@ public function __construct() {
 		add_action( 'sensei_learner_profile_info', array( $this, 'learner_profile_courses_heading' ), 30, 1 );
 
 		// Add class to body tag
-		add_filter( 'body_class', array( $this, 'learner_profile_body_class' ), 10, 1 );
+		add_action( 'wp_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
 	} // End __construct()
 
+	/**
+	 * Enqueue frontend JavaScripts.
+	 *
+	 * @since  1.0.0
+	 * @return void
+	 */
+	public function enqueue_scripts() {
+		$disable_js = Sensei_Utils::get_setting_as_flag( 'js_disable', 'sensei_settings_js_disable' );
+
+		if ( ! $disable_js ) {
+
+			$suffix = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
+
+			// My Courses tabs script.
+			wp_register_script( Sensei()->token . '-user-dashboard', esc_url( Sensei()->plugin_url . 'assets/js/user-dashboard' . $suffix . '.js' ), array( 'jquery-ui-tabs' ), Sensei()->version, true );
+			wp_enqueue_script( Sensei()->token . '-user-dashboard' );
```

Comment:
Out of scientific interest, why separate `wp_register_script()`? Is it ever going to be enqueued from anywhere else?

Reply:
I'm not very qualified to answer this, since I'm not very familiar with the different methods and pro's / con's of each ... that is how the original code was, when I pulled it from the `frontend.php` file. If you think it's better to pull out the `wp_register_script()` call and only call `wp_enqueue_script()`, I can definitely change that.

Reply:
@donnapep thoughts? Any conventions to follow vs actual need?

Reply:
`wp_enqueue_script` seems sufficient here.